File: ecopymobile/myapp/lib/model/request.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'request.g.dart';

@JsonSerializable()
class Request {
  String? id;
  int? status;
  int? collatedPrintOptionId;
  int? orientationId;
  int? letterId;
  int? pagePerSheetId;
  int? printPageOptionId;
  int? sidePrintOptionId;
  double? price;
  String? comment;
  bool? isPaid;

  Request() {}

  factory Request.fromJson(Map<String, dynamic> json) =>
      _$RequestFromJson(json);

  /// Connect the generated [_$RequestToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$RequestToJson(this);
}

File: eCopy.Services/Database/Request.cs
using eCopy.Services.Database;
using System;
using System.Collections.Generic;

namespace eCopy.Services
{
    public partial class Request
    {
        public int Id { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? ModifiedDate { get; set; }
        public bool? Active { get; set; }
        public string Status { get; set; } = null!;
        public string? FilePath { get; set; }


        public int CollatedPrintOptionId { get; set; }
        public CollatedPrintOption CollatedPrintOption { get; set; }

        public int OrientationId { get; set; }
        public Orientation Orientation { get; set; }

        public int LetterId { get; set; }
        public Letter Letter { get; set; }

        public int PagePerSheetId { get; set; }
        public PagePerSheet PagePerSheet { get; set; }

        public int PrintPageOptionId { get; set; }
        public PrintPageOption PrintPageOption { get; set; }

        public int SidePrintOptionId { get; set; }
        public SidePrintOption SidePrintOption { get; set; }


        public int ClientId { get; set; }
        public int CopierId { get; set; }
        public DateTime RequestDate { get; set; }
        public double Price { get; set; }
        public bool IsPaid { get; set; }
        public string Comment { get; set; }

        public virtual Client Client { get; set; } = null!;
        public virtual Copier Copier { get; set; } = null!;
    }
}
